From b2e05bfdcdc5b07d3115b8928e4727472dc4bb79 Mon Sep 17 00:00:00 2001
From: Benedikt Morbach <benedikt.morbach@googlemail.com>
Date: Wed, 16 Mar 2011 19:29:51 +0100
Subject: [PATCH] create cache under /var, not ${PREFIX}/var

Upstream: Submitted, http://www.bughost.org/pipermail/power/2011-March/002160.html

The cache was created under ${PREFIX}/var/cache/powertop,
when it should have been under /var/cache/powertop.

Also, this wasn't where the code was looking for it,
but the Makefile was simply wrong.
---
 Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Makefile b/Makefile
index 9399c86..5e0bf5b 100644
--- a/Makefile
+++ b/Makefile
@@ -69,7 +69,7 @@ powertop: $(OBJS) $(HEADERS)
 install: powertop
 	mkdir -p ${DESTDIR}${BINDIR}
 	cp powertop ${DESTDIR}${BINDIR}
-	mkdir -p ${DESTDIR}${PREFIX}/var/cache/powertop
+	mkdir -p ${DESTDIR}/var/cache/powertop
 	@(cd po/ && env LOCALESDIR=$(LOCALESDIR) DESTDIR=$(DESTDIR) $(MAKE) $@)
 	
 
-- 
1.7.4.1

